common/djangoapps/labster/course/views.py

```diff
@@ -306,9 +306,12 @@ def course_handler(request, course_key_string=None):
                 delete_course_and_groups(course_key, request.user.id)
                 return Response(status=status.HTTP_204_NO_CONTENT)
 
-    except LicenseStoreError:
-        raise HttpResponseBadRequest({
-            'ErrMsg': 'Incorrect license format.'
+    except LicenseStoreError as err:
+        log.exception(
+            'Incorrect licenses format: %s - %r', request.json.get('licenses', None), err
```

Comment:
I prefer to create before message and than use it in exceptions. 
Like: 
msg = 'Incorrect licenses format: %s - %r', request.json.get('licenses', None)
log.exception(msg), err
return HttpResponseBadRequest({"ErrMsg": msg})